src/main/resources/db/migration/V1_9__Project.sql

```diff
@@ -0,0 +1,9 @@
+create table  PROJECT
+(
+    ID bigint not null constraint PK_PROJECT primary key,
+    NAME varchar (101) unique not NULL ,
```

Comment:
why 101 and not 100?

Reply:
size uses <= while varchar uses < to determin the length, i discovered it by testing